### PR TITLE
Add documentation page using nextra

### PR DESCRIPTION
**Pull request by nkilm**
## Related Issues
#144 

## Changes
- New documentation page using [Nextra](https://nextra.site/)

I've set up a basic documentation site using the guide mentioned the official nextra site and starter template. We can improve upon this. Nextra covers most of the usecases i.e, syntax highlighting, cmd + k, table of contents, etc.

## References
- https://nextra.site/docs/docs-theme
- https://github.com/shuding/nextra-docs-template/

Any suggestions are welcome!

## Testing locally
cd into `/app/docs` and install using `pnpm i`. Later start the server using `pnpm dev`. It starts the server on PORT 5000.

## Visuals

image

image

### CODE REVIEW

File: apps/docs/.gitignore
Comment (thibaultleouay) [inline apps/docs/.gitignore:1]: can't we use the main .gitignore ?

File: apps/docs/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "start": "next start"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openstatusHQ/openstatus"
+  },
+  "homepage": "https://openstatus.dev",
+  "dependencies": {
+    "next": "^13.4.12",
+    "nextra": "latest",
```
Comment (thibaultleouay) [inline apps/docs/package.json]: instead of latest can we specify the version to avoid some issue if there's an update ?

File: apps/docs/.gitignore
Comment (nkilm) [reply apps/docs/.gitignore:1]: yes we can. I thought it was kind of like a separate project so I left it like that.

File: apps/docs/package.json (hunk 1 above)
Comment (nkilm) [reply apps/docs/package.json]: Alright.

File: apps/docs/tsconfig.json
Comment (thibaultleouay) [inline apps/docs/tsconfig.json:1]: we should also extend our own tsconfig 

in dev dependencies add "tsconfig":"workspace:*"

File: apps/docs/package.json (hunk 1 above)
Comment (nkilm) [reply apps/docs/package.json]: `2.10.0` appears to be the latest version, I'll add that.

File: apps/docs/tsconfig.json
Comment (nkilm) [reply apps/docs/tsconfig.json:1]: Alright

**Comment by thibaultleouay** [comment]
@nkilm  thanks for the work will merge it 🚀

**Pull request merged**